vermouth/data/force_fields/amber/aminoacids.rtp

```diff
@@ -0,0 +1,3460 @@
+[ bondedtypes ]
+; Column 1 : default bondtype
+; Column 2 : default angletype
+; Column 3 : default proper dihedraltype
+; Column 4 : default improper dihedraltype
+; Column 5 : This controls the generation of dihedrals from the bonding.
+;            All possible dihedrals are generated automatically. A value of
+;            1 here means that all these are retained. A value of
+;            0 here requires generated dihedrals be removed if
+;              * there are any dihedrals on the same central atoms
+;                specified in the residue topology, or
+;              * there are other identical generated dihedrals
+;                sharing the same central atoms, or
+;              * there are other generated dihedrals sharing the
+;                same central bond that have fewer hydrogen atoms
+; Column 6 : number of neighbors to exclude from non-bonded interactions
+; Column 7 : 1 = generate 1,4 interactions between pairs of hydrogen atoms
+;            0 = do not generate such
+; Column 8 : 1 = remove proper dihedrals if found centered on the same
+;                bond as an improper dihedral
+;            0 = do not generate such
+; bonds  angles  dihedrals  impropers all_dihedrals nrexcl HH14 RemoveDih
+     1       1          9          4        1         3      1     0
+
+; now: water, ions, urea, terminal caps, AA's and terminal AA's
+
+; tip3p
+[ HOH ]
+ [ atoms ]
+    OW   OW           -0.834    0
+   HW1   HW            0.417    0
+   HW2   HW            0.417    0
+ [ bonds ]
+    OW   HW1
+    OW   HW2
+
+; tip4p
+[ HO4 ]
+ [ atoms ]
+    OW   OW_tip4p      0.00     0
+   HW1   HW            0.52     0
+   HW2   HW            0.52     0
+    MW   MW           -1.04     0
+ [ bonds ]
+    OW   HW1
+    OW   HW2
+
+[ IB+ ] ; big positive ion
+ [ atoms ]
+   IB     IB           1.00000     1
+
+[ CA ]
+ [ atoms ]
+   CA     C0           2.00000     1
+
+[ CL ]
+ [ atoms ]
+   CL     Cl          -1.00000     1
+
+[ NA ]
+ [ atoms ]
+   NA     Na           1.00000     1
+
+[ MG ]
+ [ atoms ]
+   MG     MG           2.00000     1
+
+[ K ]
+ [ atoms ]
+   K      K            1.00000     1
+
+[ RB ]
+ [ atoms ]
+   RB     Rb           1.00000     1
+
+[ CS ]
+ [ atoms ]
+   CS     Cs           1.00000     1
+
+[ LI ]
+ [ atoms ]
+   LI     Li           1.00000     1 
+
+[ ZN ]
+ [ atoms ]
+   ZN     Zn           2.00000     1
```

Comment:
Do we want to keep water and ions? Same question goes for everything that's not an amino acid (URE, ACE, ...)

Reply:
I'll remove that stuff. I didn't think about it when copying the GMX file to be honest